<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ERTUi</class>
 <widget class="QDialog" name="ERTUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ertBG" >
     <property name="title" >
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
     <item>
       <widget class="QRadioButton" name="collapsedRB" >
        <property name="toolTip" >
         <string>Show ERT button only</string>
        </property>
        <property name="text" >
         <string>&amp;Collapsed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="openRB" >
        <property name="toolTip" >
         <string>Show ERT contents</string>
        </property>
        <property name="text" >
         <string>O&amp;pen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
